src/User32.Desktop/User32+INPUT.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="INPUT"/> nested type and its companion types <see cref="InputType"/> and <see cref="InputUnion"/>.
```

Comment:
I see you split it out into another file, but this comment still needs to get updated.